### ISSUE TITLE
DefineGrid: Indexing error

### ISSUE THREAD

**Issue by braymp** [CellProfiler/CellProfiler#3331]
The attached 3.0rc5 pipeline produces an error when defining a grid:
```
Failed to run module DefineGrid
Traceback (most recent call last):
  File "cellprofiler\gui\pipelinecontroller.py", line 2887, in do_step
    self.__pipeline.run_module(module, workspace)
  File "cellprofiler\pipeline.py", line 2022, in run_module
    module.run(workspace)
  File "cellprofiler\modules\definegrid.py", line 427, in run
    background_image = self.get_background_image(workspace, gridding)
  File "cellprofiler\modules\definegrid.py", line 477, in get_background_image
    gridding.x_spacing / 2) * 2 + 2, 3))
TypeError: 'float' object cannot be interpreted as an index
```
[issue.zip](https://github.com/CellProfiler/CellProfiler/files/1353739/issue.zip)

**Comment by 0x00b1** [comment]
Sentry issue: https://sentry.io/allen-goodman/cellprofiler/issues/361566130/

**Comment by mcquin** [comment]
Resolved #3443
